manual_flagging/flagging.py

```diff
@@ -94,6 +94,19 @@
     WHERE condo.class IN ('297', '299', '399')
     AND NOT condo.is_parking_space
     AND NOT condo.is_common_area
+),
+
+-- Select neighborhood groups and filter for most recent versions
+neighborhood_group AS (
+    SELECT nbhd_group.nbhd, nbhd_group.group_name
+    FROM location.neighborhood_group AS nbhd_group
+    INNER JOIN (
+        SELECT nbhd, MAX(version) AS version
+        FROM location.neighborhood_group
+        GROUP BY nbhd
+    ) AS latest_group_version
```

Comment:
Note that this join could also be extended to identify the version of the neighborhood groups that was used for a specific run; see the removed code in 70af4829de14313652ead423b5dc6c007da53aca for a sketch of what that would look like.